x-pack/plugins/maps/public/classes/styles/vector/components/symbol/icon_stops.js

```diff
@@ -43,13 +43,25 @@ export function getFirstUnusedSymbol(iconStops) {
   return firstUnusedSymbol ? firstUnusedSymbol.value : DEFAULT_ICON;
 }
 
-export function IconStops({ field, getValueSuggestions, iconStops, onChange }) {
-  return iconStops.map(({ stop, icon }, index) => {
-    const onIconSelect = (selectedIconId) => {
+export function IconStops({
+  field,
+  getValueSuggestions,
+  iconStops,
+  onChange,
+  onCustomIconsChange,
+  customIcons,
+}) {
+  return iconStops.map(({ stop, value }, index) => {
+    const { svg, label } = value.startsWith(CUSTOM_ICON_PREFIX_SDF)
```

Comment:
`value.startsWith(CUSTOM_ICON_PREFIX_SDF)` is a little hacky. Would it be possible to store state in IconStop type that indicates if icon is maki or custom instead of interrogating id? Maybe we could add `iconSource` property to IconStop that would be either MAKI or CUSTOM?